src/function.js

```diff
@@ -260,6 +260,19 @@ Sk.builtin.func = function (code, globals, closure, closure2) {
     if (code.co_fastcall) {
         this.tp$call = code;
     }
+    // Because our external API allows you to set these flags
+    // *after* constructing the function (grr), we can only
```

Comment:
should we make them part of the `Sk.func` constructor instead of plonking them on the prototype of `func_code`?

Reply:
Yes, we should! That will make constructing functions from JS much nicer, too. But again, it's an API-breaking change, so I'm leaving it out of this PR.